packages/client/ui/react-ui/README.md

```diff
@@ -29,11 +29,13 @@ We provide two essential React context providers:
 
 - `CrossmintProvider` - Base provider for accessing Crossmint services
 - `CrossmintAuthProvider` - Authentication context provider for your components
+- `CrossmintWalletProvider` - Wallet context provider for your components, can be used to integrate with other auth providers. Visit [Crossmint Signers Demo](https://crossmint-signers-demo.vercel.app/examples/dynamic) for complete implementation details and advanced features.
 
 #### Supported Login Methods
 - **Email OTP**: Passwordless sign-in using a one-time code
-- **Social Accounts**: Sign in with Google, Apple, X, and more
+- **Social Accounts**: Sign in with Google, X, and more
 - **Farcaster**: Using the Sign In With Farcaster (SIWF) standard
+- **Web3**: Sign in with EOA wallet (limited support for now) but will be available soon
```

Comment:
would remove "but will be available soon". and we have auth working anyway so...